client-app/shared/layout/components/search-bar/search-bar.vue

```diff
@@ -29,6 +29,39 @@
       >
         <!-- Results -->
         <template v-if="categories.length || products.length">
```

Comment:
`v-if="categories.length || products.length || suggestions.length || pages.length"`
or for example
`v-if="isExistResults"`
`...`
`const isExistResults = computed(() => categories.value.length || products.value.length || suggestions.value.length || pages.value.length)`